lib/fetch.spec.js

```diff
@@ -85,8 +87,17 @@ describe('Data sanitation', () => {
 
     expect(sanitizeEventAndNews(event)).to.deep.equal(emptyResponse);
   });
+
+
+  it('should copy start datetime to end datetime when end datetime is empty', () => {
+    const response = sanitizeEventAndNews(prismicEventResponse, 'event');
+
+    expect(response.startDateTime).to.deep.equal(response.endDateTime);
+    expect(response.endDateTime).to.equal('2016-07-27T23:00:00+0000');
+  });
```

Comment:
oh do you mean a test to compare parsed structure of the startDateTime {} equals endDateTime {} ?